frontend/src/pages/lessonsPage/Question.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { Text, Center, Box, Stack, SimpleGrid, Button } from '@chakra-ui/react';
+
+const Question = (props) => {
```

Comment:
De-structure your passed props para di na mag `props.{passedProps}`
```suggestion
const Question = ({updateProgress}) => {
```